### PR TITLE
fix:(DB\Smart_Scripts):Thundering Invaders should not despawn

### PR DESCRIPTION
**Pull request by Ragelezz**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7417
- Closes https://github.com/chromiecraft/chromiecraft/issues/1456

## Tests Performed:

- .go c id 14462 and it despawned either after 500ms (when out of combat) either 2 mins after spawned.
- 

## How to Test the Changes:

1. .go c id 14462 and look for the portal, stay near portal and look at elementals, they won't despawn anymore
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1632568882031573700.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632568882031573700');
+
+UPDATE `smart_scripts` SET `action_param3`=990000 WHERE  `entryorguid`=179664 AND `source_type`=1 AND `id`=0 AND `link`=0;
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=14462;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1632568882031573700.sql:5]: miss source_type here, you need to delete per entryorguid and per source_type, so add AND source_type = 0

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1632568882031573700.sql:5]: @Ragelezz

Comment (Ragelezz) [reply data/sql/updates/pending_db_world/rev_1632568882031573700.sql:5]: after irl work

**Comment by stoudtlr** [comment]
> Doesn't look like it should. (See my comment at #8082 )

I couldn't remember what the event was like so I only checked that they didn't despawn since that's what the PR said.  Agree with you after checking the video you linked.

It looks like each portal should only spawn 1 at a time instead of continuously spawning them.

**Comment by temperrr** [comment]
@Helias 

I think the current behaviour (without PR) is better than this, as having too many spawned can cause economic/too fast leveling issues.

**Comment by Ragelezz** [comment]
So...

**Comment by Ragelezz** [comment]
@azerothcore/testers

**Comment by FrancescoBorzi** [comment]
@temperrr what's the status of this PR?

**Comment by temperrr** [comment]
Close this PR, needs more research to find out how it should be.

**Pull request closed without merging**